### DISCUSSION TITLE
How to set TEMPDIR

### DISCUSSION THREAD

**Discussion by FlorianEndel** [cupcakearmy/autorestic#232] [Q&A] [answered]
Is there a possibility to set the TEMPDIR of restic with autorestic?

Reference in the restic-manual: `https://restic.readthedocs.io/en/latest/manual_rest.html?highlight=TMPDIR#temporary-files`

**Comment by rdelaage** [comment, marked as answer]
Not in the config file I think. May be you can open a feature request for this

**Comment by FlorianEndel** [comment]
It is not possible to set environment variables per location but it is possible to set them per backend https://autorestic.vercel.app/backend/env#example-with-backblaze-b2
